### PR TITLE
date-format in axis

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/resources/config/liquibase/changelog/20200704100000_added_dateformate_property.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet author="khushbu" id="20200704100000">
```
Comment (johibkhan2) [inline src/main/resources/config/liquibase/changelog/20200704100000_added_dateformate_property.xml:3]: Time stamp date is 04 july.. It should be latest date

File: src/main/resources/config/liquibase/master.xml [hunk 2]
```diff
@@ -83,6 +83,7 @@
     <include file="config/liquibase/changelog/20200705100000_aggregate_percent_function.xml" relativeToChangelogFile="false"/>
     <include file="config/liquibase/changelog/20200707100000_update_column_propertytype_iframe_setting.xml" relativeToChangelogFile="false"/>
 
+    <include file="config/liquibase/changelog/20200704100000_added_dateformate_property.xml" relativeToChangelogFile="false" />
```
Comment (johibkhan2) [inline src/main/resources/config/liquibase/master.xml:86]: Here also

File: src/main/webapp/app/services/visualizations/stackedverticalbar-chart.service.js [hunk 3]
```diff
@@ -44,6 +44,7 @@
                     result['axisScaleLabel'] = VisualizationUtils.getPropertyValue(record.properties, 'Axis Scale Label');
                     result['showLegend'] = VisualizationUtils.getPropertyValue(record.properties, 'Show Legend');
                     result['legendPosition'] = VisualizationUtils.getPropertyValue(record.properties, 'Legend position').toLowerCase();
+                    result['dateFormate'] = VisualizationUtils.getPropertyValue(record.properties, 'Datetime format');
```
Comment (johibkhan2) [inline src/main/webapp/app/services/visualizations/stackedverticalbar-chart.service.js:47]: This text used 4 times.. Create a const

Comment (khushbu-maurya) [reply src/main/webapp/app/services/visualizations/stackedverticalbar-chart.service.js:47]: for every configuration, we are directly using the name of the property. 
we are planning to create common servicer to process metadata 
that we can use in flair-bi and flair-notification
at that time we can create  const file for property name 
leave it for now

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Pull request merged**